opal/browser/form_data.rb

```diff
@@ -172,7 +172,13 @@ def set(key, value, filename = nil)
 
   # Iterate over all elements of this FormData
   def each(&block)
-    Native(`#@native.getAll()`).each(&block)
+    hash = {}
+    %x{
+      for (var pair of #@native.entries()) {
+        #{hash[`pair[0]`] = `pair[1]`}
+      }
+    }
+    hash.each(&block)
```

Comment:
Since we're iterating we could just yield the two pairs without creating a supporting hash, that way we'd also get the advantage of supporting duplicate keys (which are a thing IIRC)